server/ServerFSM.cpp

```diff
@@ -1273,11 +1273,40 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
             psd.second.m_save_game_empire_id = ALL_EMPIRES;
         }
 
+        // remove all AIs
```

Comment:
clarify: from current lobby data, so that when the save is loaded they can be re-added from a fresh / no AI state as appropriate, without having potential extra AIs lingering from the previous / abandoned game setup...

Reply:
Fixed.